docsTech/filtering.md

```diff
@@ -0,0 +1,38 @@
+# Filtering
+
+## Structure
+
+### Data
+
+First, let's see how the filtering is saved in the state. Here are the types of the filter model. It is made of a `linkOperator`, and a list of items. Each item corresponds to a line in the filtering panel. Each item corresponds to a line in the filter panel.
+
+```ts
+interface GridFilterModel {
```

Comment:
For the `Data`, I would just link to the doc api pages to avoid outdated information here.